src/validation.h

```diff
@@ -849,13 +850,15 @@ class ChainstateManager
     bool AcceptBlockHeader(
         const CBlockHeader& block,
         BlockValidationState& state,
-        const CChainParams& chainparams,
         CBlockIndex** ppindex) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
     friend CChainState;
 
 public:
     explicit ChainstateManager(const CChainParams& chainparams) : m_chainparams{chainparams} { }
 
+    const CChainParams& GetParams() const { return m_chainparams; }
+    const Consensus::Params& GetConsensus() const { return m_chainparams.GetConsensus(); }
+
```

Comment:
Perhaps these additions could be made in the previous commit where we introduce the private `m_chainparams` member